lib/diaspora_federation/validators/relayable_validator.rb

```diff
@@ -0,0 +1,11 @@
+module DiasporaFederation
+  module Validators
+    module RelayableValidator
+      def self.included(model)
+        model.class_eval do
+          rule :parent_guid, :guid
```

Comment:
@jhass, could you please check this outcomes? If it is a bug, we need to find out, how it is supposed to work.